cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala

```diff
@@ -40,6 +42,7 @@ class CallTest extends DocumentingTest {
         |* <<call-call-a-procedure-with-parameter-arguments, Call a procedure with parameter arguments>>
         |* <<call-call-a-procedure-with-mixed-literal-and-parameter-arguments, Call a procedure with mixed literal and parameter arguments>>
         |* <<call-call-a-procedure-with-literal-and-default-arguments, Call a procedure with literal and default arguments>>
+        |* <<call-call-a-procedure-call-yield-star, Call a procedure using `CALL YIELD *`>>
```

Comment:
Escape the inline Cypher query.

```
|* <<call-call-a-procedure-call-yield-star, Call a procedure using `+CALL YIELD *+`>>
```

Reply:
just need to confirm that they are just escaping and not being printed (have had places before where they were printed and had to be removed)

Reply:
https://github.com/neo-technology/neo4j-manual-modeling/wiki/Asciidoc-examples#cases-when-to-escape-cypher-queries

Reply:
I'm guessing its fine, I know places we have needed the escaping and places where we found random `+` had appeared, I have updated to escape them, and it would be good to build and look to make sure it works as intended :)